### PR TITLE
Models

### PR DESCRIPTION
**Pull request by allenaerostar**
Barebone username, email, and password account creation endpoints created with input validations.

### CODE REVIEW

File: dietstory-routing/registration/validate_forms.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+from django import forms
+from datetime import date
+from django.core.validators import validate_email, validate_slug, BaseValidator
+
+
+def calculate_age(birthday):
+    today = date.today()
+    return today.year - birthday.year - ((today.month, today.day) < (birthday.month, birthday.day))
+
```
Comment (BenjixD) [inline dietstory-routing/registration/validate_forms.py]: make `calculate_age` a member function (or static function of) of `MinAgeValidator` if only used there

File: dietstory-routing/registration/views.py [hunk 2]
```diff
@@ -1,16 +1,35 @@
...
+        return HttpResponse("Welcome to signup page. Please enter username, email, password, and birthday.", status=200)
+
+    # @method_decorator(sensitive_post_parameters('password'))
+    def post(self, request, *args, **kwargs):
 
-class AccountsViewSet(viewsets.ModelViewSet):
-    queryset = Accounts.objects.all().order_by('name')
-    serializer_class = AccountSerializer
+        params = SignupForm(request.data)
+        print((params))
```
Comment (BenjixD) [inline dietstory-routing/registration/views.py]: debug log to be removed?

File: dietstory-routing/registration/views.py [hunk 3]
```diff
@@ -1,16 +1,35 @@
...
+        if params.is_valid():
+            username = params.cleaned_data.get('username')
+            accounts = Accounts.objects.filter(name=username)
+            if not accounts:
+                password = params.cleaned_data.get('password1')
+                email = params.cleaned_data.get('email')
+                birthday = params.cleaned_data.get('birthday')
+                account = Accounts(name=username, password=password, email=email, birthday=birthday)
```
Comment (BenjixD) [inline dietstory-routing/registration/views.py]: Can this fail? If it does, what happens?

What if password != params.cleaned_data.get('password1')? Are we creating an account with a non-deterministic username?

File: dietstory-routing/registration/views.py (hunk 2 above)
Comment (allenaerostar) [reply dietstory-routing/registration/views.py]: yes for sure.

File: dietstory-routing/registration/validate_forms.py (hunk 1 above)
Comment (allenaerostar) [reply dietstory-routing/registration/validate_forms.py]: Was planning to make that function used by more than one class, but for now MinAgeValidator is the only one using it. calculate_age is moved into the MinAgeValidator class now.

**Pull request merged**